### PR TITLE
include instrs to restart db servers for self-hosted in discover

### PR DESCRIPTION
**Pull request by stevenGravy**
Include instructions to restart the database servers to apply configurations. This is included for mongodb but not other dbs.

### CODE REVIEW

File: web/packages/teleport/src/Discover/Database/MutualTls/MutualTls.tsx [hunk 1]
```diff
@@ -156,6 +156,9 @@ function DbEngineInstructions({ dbEngine }: { dbEngine: DatabaseEngine }) {
             },
           ]}
         />
+        <Text mb={1}>
```
Comment (kimlisa) [inline web/packages/teleport/src/Discover/Database/MutualTls/MutualTls.tsx:159]:
```suggestion
        <Text mt={1}>
```

File: web/packages/teleport/src/Discover/Database/MutualTls/MutualTls.tsx [hunk 2]
```diff
@@ -244,6 +247,9 @@ function DbEngineInstructions({ dbEngine }: { dbEngine: DatabaseEngine }) {
                       },
                     ]}
                   />
+                  <Text mb={1}>
```
Comment (kimlisa) [inline web/packages/teleport/src/Discover/Database/MutualTls/MutualTls.tsx:250]:
```suggestion
                  <Text mt={1}>
```

File: web/packages/teleport/src/Discover/Database/MutualTls/MutualTls.tsx [hunk 3]
```diff
@@ -274,6 +280,9 @@ function DbEngineInstructions({ dbEngine }: { dbEngine: DatabaseEngine }) {
...
```
Comment (kimlisa) [inline web/packages/teleport/src/Discover/Database/MutualTls/MutualTls.tsx:283]:
```suggestion
                  <Text mt={1}>
```

**Review by kimlisa — APPROVED** [review]
ty!

**Review by marcoandredinis — APPROVED** [review]

**Pull request merged**